spec/helpers/application_formatters_helper_spec.rb

```diff
@@ -63,6 +65,23 @@
         expect(helper.format_html(html)).to eq(html)
       end
 
+      it 'does not remove image src' do
```

Comment:
Let's group up the image tags in a context? Easier to find and replace if we need to.